test/integration/install_scripts_int_test.go

```diff
@@ -260,6 +264,21 @@ func (suite *InstallScriptsIntegrationTestSuite) assertCorrectVersion(ts *e2e.Se
 	}
 }
 
+func (suite *InstallScriptsIntegrationTestSuite) assertAnalytics(ts *e2e.Session) {
+	// Verify analytics reported the correct sessionToken.
+	sessionTokenFound := false
+	events := parseAnalyticsEvents(suite, ts)
+	suite.Require().NotEmpty(events)
+	for _, event := range events {
+		if event.Category == anaConst.CatInstallerFunnel && event.Dimensions != nil {
+			suite.Assert().NotEmpty(*event.Dimensions.SessionToken)
```

Comment:
nit: We _could_ assert the equality of the token value, not just is presence, as the comment above suggests.

Reply:
I've updated the comment. I am guilty of copy-pasting the code from another test :/ The important thing is that it's not empty.